src/main/java/duke/commands/Command.java

```diff
@@ -0,0 +1,23 @@
+package duke.commands;
+
+import duke.exceptions.DukeException;
+import duke.storage.Storage;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+public class Command {
```

Comment:
Maybe consider making this abstract?
```suggestion
public abstract class Command {
```